### ISSUE TITLE
Contributor to vroom

### ISSUE THREAD

**Issue by jonathf** [VROOM-Project/vroom#644]
To be able to experiment with new pyvroom features I sometimes need to make changes to vroom itself.
But if I don't have push-access to vroom, I will still need to use my own fork.

Would it be possible to get contributor access to the vroom repo?

**Comment by jcoupey** [comment]
Let me rephrase to make sure I understand the problem: you're currently using your own fork as a git submodule but if you transfer `pyvroom` here and point the submodule to the upstream `vroom` repo, then you're stuck.

In that case, what would be the typical upstream workflow? Something like:
- pushing a branch in `vroom`
- pointing the `pyvroom` submodule to that branch for testing
- drafting a PR if it makes sense to include the changes upstream

**Comment by jonathf** [comment]
Yes, exactly.

**Comment by jcoupey** [comment]
I just sent the invite for Write access to the `vroom` core repo.
